src/main/java/service/operation/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.operation;
+
+import java.util.Map;
+import model.FruitTransaction;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
Remember to follow the encapsulation principle. Consider if the visibility of the class and its members is appropriate, and whether some members could be more tightly encapsulated.